src/endpoints/boundary_event/boundary_event_socket_endpoint.ts

```diff
@@ -0,0 +1,93 @@
+import {Logger} from 'loggerhythm';
+
+import {UnauthorizedError} from '@essential-projects/errors_ts';
+import {IEventAggregator, Subscription} from '@essential-projects/event_aggregator_contracts';
+import {BaseSocketEndpoint} from '@essential-projects/http_node';
+import {IIdentity, IIdentityService} from '@essential-projects/iam_contracts';
+
+import {Messages, socketSettings} from '@process-engine/management_api_contracts';
+
+const logger: Logger = Logger.createLogger('management_api:socket.io_endpoint:user_tasks');
```

Comment:
Please make use of type-inference.

```suggestion
const logger = Logger.createLogger('management_api:socket.io_endpoint:user_tasks');
```